framework/packages/abstract-std/src/objects/account/account_trace.rs

```diff
@@ -174,12 +174,13 @@ impl AccountTrace {
         let acc = if trace == LOCAL {
             Self::Local
         } else {
-            Self::Remote(
-                trace
-                    .split(CHAIN_DELIMITER)
-                    .map(TruncatedChainId::_from_str)
-                    .collect(),
-            )
+            let rev_trace: Vec<_> = trace
```

Comment:
What about the `from_string` method lines 155-167?

Reply:
(a method which also needs a test case)

Reply:
Put things in common, test case covers everything